ipatool

```diff
@@ -115,6 +116,7 @@ browser: firefox
 # GitHub configuration
 gh-token: "0123456789abcdef0123456789abcdef01234567"
 gh-repo: "freeipa/freeipa"
+gh-fork-remote: "ghfreeipa"
```

Comment:
Please use `"my-github"` or something similar as an example. This can be confused as "github freeipa" (as in mirror of upstream pagure), which is not what we want here.